packages/e2e-test-utils/src/activatePlugin.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { setCurrentUser } from './user';
+import visitAdminPage from './visitAdminPage';
+
+async function activatePlugin(slug) {
+  await setCurrentUser('admin', 'password');
+  await visitAdminPage('plugins.php');
+  const disableLink = await page.$(`tr[data-slug="${slug}"] .deactivate a`);
+
+  if (disableLink) {
+    await setCurrentUser('admin', 'password');
```

Comment:
That does nothing..?

The user is already an admin at this point.

I think what we'd want in this util is:

```js
const currentUser = getCurrentUser();
await setCurrentUser('admin', 'password');

if ( can activate plugin ) {
  // activate plugin
}

await setCurrentUser(currentUser.username, currentUser.password);
```

Reply:
Copied from 
https://github.com/WordPress/gutenberg/blob/2826f5005df7f8e303730dc79aded9e5ea890fd5/packages/e2e-test-utils/src/deactivate-plugin.js#L16-L20

I dont really know what this does.

Reply:
In Gutenberg that does:

1. Switch to admin
2. Go to plugins page
3. -> If plugin is already deactivated, switch back to user being tested, and do nothing
4. Deactivate plugin
4. Switch back to user being tested

Similar for plugin activation.

So what we want is roughly what I pasted above, essentially the same logic.

Just not two `setCurrentUser('admin', 'password')` in a row.

Reply:
```js
/**
 * Internal dependencies
 */
import { getCurrentUser, setCurrentUser } from './user';
import visitAdminPage from './visitAdminPage';

async function activatePlugin(slug) {
  const currentUser = getCurrentUser();
  await setCurrentUser('admin', 'password');

  await visitAdminPage('plugins.php');
  const disableLink = await page.$(`tr[data-slug="${slug}"] .deactivate a`);

  if (!disableLink) {
    await page.click(`tr[data-slug="${slug}"] .activate a`);
    await page.waitForSelector(`tr[data-slug="${slug}"] .deactivate a`);
  }

  await setCurrentUser(currentUser.username, currentUser.password);
}

export default activatePlugin;
```